users/serializers.py

```diff
@@ -83,23 +86,35 @@ def __init__(self, **kwargs: Any) -> None:
     )
 
     def validate_email(self, email: str) -> str:
+        logger.info("Custom register serializer - validate email method")
         email = allauth_get_adapter().clean_email(email)
         custom_validate_email_function(self, email)
         if allauth_settings.UNIQUE_EMAIL:
             if email and email_address_exists(email):
+                logger.warning(
+                    f"Email: {email} is already exist and throw exception with message: {CustomValidationErrorText.VALIDATION_ERROR_SIGNUP_EMAIL_MESSAGE} "
```

Comment:
`{email} already exists`. The exception message is constant, we don't need it.

Reply:
Done